src/breakpoints.jl

```diff
@@ -7,6 +7,21 @@ Return an array with all breakpoints.
 """
 breakpoints() = _breakpoints
 
+"""
+    breakpoint_update_hooks
+
+This is a collection of "hook" functions, each taking as input function, and a `AbstractBreakpoint`.
```

Comment:
This section needs proofreading. I think structuring the API similar to the `atreplinit` function is a good idea.

Also, please add that this feature is experimental.

Reply:
Sounds good, I just took a look at that code,
I will make that change tomorrow.

Reply:
done